src/providers/export/cntk.ts

```diff
@@ -76,13 +76,12 @@ export class CntkExportProvider extends ExportProvider<ICntkExportProviderOption
             });
         });
 
+        const buffer = await HtmlFileReader.getAssetArray(assetMetadata.asset);
```

Comment:
The `HtmlFileReader` updates the incoming asset and appends a .jpg to the name.  Not the best practice but it's consistent with the other exporters.